diff -urN tkgate-1.3b.orig/config.h tkgate-1.3b/config.h
--- tkgate-1.3b.orig/config.h	Thu May  4 20:50:25 2000
+++ tkgate-1.3b/config.h	Thu May 11 11:20:19 2000
@@ -73,7 +73,7 @@
   compile flags to use for gcc.  If you change this, you will
   need to run the configure script again.
  */
-#define TKGATE_GCCFLAGS		-O2
+#define TKGATE_GCCFLAGS		$(OPT_FLAGS)
 
 /*
   compile flags to use for cc.  If you change this, you will
diff -urN tkgate-1.3b.orig/sitename.txt tkgate-1.3b/sitename.txt
--- tkgate-1.3b.orig/sitename.txt	Fri Apr 14 04:33:07 2000
+++ tkgate-1.3b/sitename.txt	Thu May 11 10:07:39 2000
@@ -1 +1 @@
-Earth
+On Earth, with PLD Linux
